### DISCUSSION TITLE
What makes MinIO the most used solution?

### DISCUSSION THREAD

**Discussion by AmirMohG** [minio/minio#15871] [Q&A] [answered]
Hello there.
After going through the MinIO documentation and actually trying the object storage I found it a decent choice to implement for our object storage service. But I recently came across Openstack Swift, which seemed just as good, which to my surprise hasn't got as much attention. could I get a brief explanation or some link to an article (I couldn't find a valid article on this topic) as to why MinIO is the superior solution here? If my question is too off-topic by any means, please redirect me to the correct discussion.

**Comment by harshavardhana** [comment, marked as answer]
If this is a production use case you can reach out to us at https://min.io/pricing

**Reply by AmirMohG** [reply]
I'm using the on-premise version.

**Reply by harshavardhana** [reply]
On premises version is nothing different, if you are a for profit organization using MinIO for profit - please read about AGPLv3 compliance requirement.
